app/assets/javascripts/discourse/app/lib/utilities.js

```diff
@@ -80,23 +80,13 @@ export function highlightPost(postNumber) {
   }
 
   const element = container.querySelector(".topic-body, .small-action-desc");
-  if (!element || element.classList.contains("highlighted")) {
+  if (!element) {
     return;
   }
 
-  element.classList.add("highlighted");
-
   if (postNumber > 1) {
-    element.setAttribute("tabindex", "0");
```

Comment:
IIRC, this is here for A11Y purposes... I guess we're removing it since we're ermoving the `removeHighlighted` function, but this would regress for A11Y. The selected post won't have focus. 

Some context in https://github.com/discourse/discourse/pull/24786

Reply:
Yeah, I removed because we'd be losing the animationend event, which would make the tabindex=0 persist forever?

What would you suggest instead? Perhaps a 1000ms setTimeout? Feels pretty hacky 😓

Reply:
Restored tabindex, with a 1000ms timeout

Reply:
Yeah, a timeout. Somewhat hacky, yes, but can't think of anything better. 

If we don't set focus at all, screen readers will start reading the page from the top, which in some cases could be many many posts away from what is visible on screen.

Reply:
Now updated to use 

```
    element.addEventListener(
      "focusin",
      () => element.removeAttribute("tabindex"),
      { once: true }
    );
```

feels a little cleaner, and seems to work well in my tests with macOS voiceover.